docs/ocean/design_docs/prognostic-sediment-transport.rst

```diff
@@ -0,0 +1,519 @@
+	/>
+  </nml_record>
+
+
+Bedload transport also requires the definitions of the following variables:
+
+- Pre-defined sediment-related variables
+	
+  * sediment grain diameter: :code:`sedimentGrainDiameter(nSedimentClasses, nCells)`
+
+  * sediment grain density: :code:`sedimentGrainDensity(nSedimentClasses, nCells)`
+
+  * sediment class fraction on bed layer: :code:`sedimentClassFraction(nSedimentClasses)`
+
+  * bedload transport coefficient: :code:`sedimentBedloadCoefficient(nSedimentClasses)`
+
+- Other variables
+	
+  * bedload transport rate: :code:`sedimentBedloadTransportRate(nCells)`
+
+
+Implementation: Suspended sediment transport
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/19
+
+Contributors: Zhendong Cao
+
+The following namelist options should be defined:
+
+- A logical swith to turn ON/OFF suspended sediment transport: :code:`config_sediment_suspended`.
+
+- A character-type variable to define the empirical suspended transport method: :code:`config_sediment_suspended_formulation`. There are three options for it:
+	
+  * :code:`None`
+
+  * :code:`Zyserman-Fredsoe`
+
+  * :code:`Goldstein`
+
+  If this is set as :code:`None`, then the suspended transport is solved by the process-based approach.
+
+- A character-type variable to define the method to compute the sediment settling velocity: :code:`config_sediment_settling_formulation`. There are three options for it:
+
+  * :code:`None`
+
+  * :code:`Soulsby`
+
+  * :code:`Cheng`
+
+  If this is set as :code:`None`, then the sediment settling velocity is pre-defined manually.
+
+These can be defined in **Registry.xml** as follows:
+
+.. code::
+
+  <nml_record name="sediment_transport" mode="init;forward">
+        <nml_option name="config_sediment_suspended" type="logical" default_value=".false." units="unitless"
+                    description="Controls if sediment bedload transport is computed."
+                    possible_values=".true. or .false."
+        />
+        <nml_option name="config_sediment_suspended_formulation" type="chracter" default_value="None" units="unitless"
+                    description="Select the sediment bedload transport formulation"
+                    possible_values=" 'None', 'Zyserman-Fredsoe', 'Goldstein'"
+	/>
+  </nml_record>
+
+Suspended sediment transport also requires the definitions of the following variables:
+
+- Pre-defined sediment-related variables
+
+  * sediment settling velocity: :code:`sedimentSettlingVelocity(nSedimentClasses)`
+
+  * sediment critical erosion shear: :code:`sedimentCriticalErosionShear(nSedimentClasses)`
+
+  * sediment erodibility constant: :code:`sedimentErodibilityConstant(nSedimentClasses)`
+
+  * sediment bed porosity: :code:`sedimentBedPorosity(nSedimentClasses)`
+
+- Other variables
+
+  * sediment suspended concentration: :code:`sedimentSuspendedConcentration(nSedimentClasses, nCell)`
```

Comment:
Again, a corresponding variable name from Algorithm Design: Suspended sediment transport would be helpful for each variable here.

The bottom velocity might warrant some discussion.  For one, the velocity magnitude isn't necessarily available at cell centers (but it can be computed from the kinetic energy).  For another, you may want to average over more than one layer to smooth out jumps depending on what vertical coordinates and bathymetries this capability supports (see my suggested requirement related to this.)